README.md

```diff
@@ -58,6 +58,19 @@ govuk-docker has the following dependencies:
 
 All other dependencies will be installed for you automatically.
 
+#### Docker settings
+These need to be changed in order to handle intensive apps like Whitehall. Docker defaults to using 2 CPU cores and 2GB of RAM. Open the Docker dropdown via the Docker whale icon in the macOS menu bar, and select the `Advanced` option.
+
+Change to ideally:
+
+* 6 CPUs
+* 12 GB RAM
+* Docker disk image size default is 64GB under the Disk tab
+  * depending on how much free space you have
+  * the minimum should be > 50GB
```

Comment:
I don't think the disk setting should be in this section since that is managed in a different option than Advanced. 

Probably worth having a section for `Advanced` and one for `Disk`. I'm also not too sure what you're saying in the Disk section. 

I'd suggest tweaking this around to be something like:

```
#### Docker Settings

Running GOV.UK applications can be resource intensive and will easily exceed the default configuration of Docker for Mac. To change settings open the Docker dropdown via the Docker whale icon in the macOS menu bar and select the preferences option. 

In `Advanced` settings you should update CPU and RAM resources. These should be at least:

* 6 CPUs
* 12 GB RAM

In `Disk` you should ensure there is a high amount of disk space to allow replicating GOV.UK data. 64GB should be sufficient for most usages but you may need > 100GB to clone all GOV.UK integration data.
```

Reply:
@kevindew Thanks Kevin! Updated in https://github.com/alphagov/govuk-docker/pull/182/commits/7d9e51df8b105471e501cae31e49a7521580377f 

And thanks for the rewrite. Not quite as as easy as hitting accept on a "Suggested commit" though 😉😂